.coveralls.yml

```diff
@@ -0,0 +1,2 @@
+service_name: travis-ci
```

Comment:
this is not necessary since you are not using a `service_job_id`
delete it
Let's just keep the `repo_token`

Reply:
Noted and rectified.